chart/kubeapps/templates/tiller-proxy-deployment.yaml

```diff
@@ -20,7 +20,8 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "kubeapps.tiller-proxy.fullname" . }}
-      terminationGracePeriodSeconds: 600
+      # Increase termination timeout to let remaining operations to finish before killing the pods
```

Comment:
I'd expand this a little bit by saying that we need a bigger terminationGracePeriod because deployments/upgrades are sync